src/fides/api/models/privacy_experience.py

```diff
@@ -141,67 +392,158 @@ def dry_update(self, *, data: dict[str, Any]) -> PrivacyExperienceConfig:
+        return (
+            self.experience_config_history.version  # type: ignore[return-value]
+            if self.experience_config_history
+            else None
+        )
+
+    @property
+    def experience_config_history(self) -> Optional[PrivacyExperienceConfigHistory]:
+        """Convenience property that returns the experience config history for the latest version.
+
+        Note that there are possibly many historical records for the given experience config translation,
+        this just returns the current corresponding historical record.
+        """
+        # Histories are sorted at the relationship level
+        return self.histories[-1] if self.histories.count() else None
+
+    @property
     def experience_config_history_id(self) -> Optional[str]:
-        """Convenience property that returns the experience config id for the current version.
+        """Convenience property that returns the experience config history id for the latest version.
 
-        Note that there are possibly many historical records for the given experience config, this just returns the current
+        Note that there are possibly many historical records for the given experience config translation, this just returns the current
         corresponding historical record.
         """
-        history: PrivacyExperienceConfigHistory = self.histories.filter_by(  # type: ignore # pylint: disable=no-member
-            version=self.version
-        ).first()
-        return history.id if history else None
-
-    @classmethod
-    def get_default_config(
-        cls, db: Session, component: ComponentType
-    ) -> Optional[PrivacyExperienceConfig]:
-        """Load the first default config of a given component type"""
         return (
-            db.query(PrivacyExperienceConfig)
-            .filter(PrivacyExperienceConfig.component == component)
-            .filter(PrivacyExperienceConfig.is_default.is_(True))
-            .first()
+            self.experience_config_history.id
+            if self.experience_config_history
+            else None
         )
 
 
-class PrivacyExperienceConfigHistory(ExperienceConfigBase, Base):
-    """Experience Config History - stores the history of how the config has changed over time"""
+class PrivacyExperienceConfigHistory(
+    ExperienceTranslationBase, PrivacyExperienceConfigBase, Base
+):
+    """Experience Config History table for auditing purposes.
 
-    experience_config_id = Column(
-        String, ForeignKey(PrivacyExperienceConfig.id_field_path), nullable=False
+    When an Experience Config and/or an Experience Translation is modified, a new version is
+    created here that stores a snapshot of these records combined.  Consent reporting can contain
+    an id to this resource which preserves the details of the Experience viewed by the end user.
+    """
+
+    custom_asset_id = Column(String, ForeignKey(CustomAsset.id_field_path))
```

Comment:
`custom_asset_id` is not a field that has been explored at all or exposed in the API, we should revisit this concept before merge.